software/auth/auth.go

```diff
@@ -173,7 +175,12 @@ func Login(domain string, oAuthOnly bool, username, password string, client hous
 		return err
 	}
 
-	workspaces, err := client.ListWorkspaces()
+	if interactive {
+		workspaces, err = client.PaginatedListWorkspaces(2, 0)
```

Comment:
Since we need to identify if the number of workspaces returned 1 or more, calling workspace pagination with page size 2 if the interactive flag is set to true